microsoft/testsuites/core/dns.py

```diff
@@ -71,15 +94,3 @@ def verify_dns_name_resolution_after_upgrade(self, node: Node) -> None:
                 expected_exit_code_failure_message="fail to run unattended-upgrade",
                 timeout=2400,
             )
-        elif isinstance(node.os, Posix):
-            node.os.update_packages("")
-        else:
-            raise UnsupportedDistroException(node.os)
-        self._check_dns_name_resolution(node)
```

Comment:
Where is this logic?

Reply:
The self._check_dns_name_resolution(node) check is moved to line #45.